Bootstrap/ARAnalyticsiOSTests/ARAnalyticsDSLTests.m

```diff
@@ -70,7 +60,7 @@ - (void)method2 {}
 
             [[[analyticsMock stub] andDo:^(NSInvocation *invocation) {
                 invocationCount++;
-            }] addEventAnalyticsHooks:OCMArg.any];
+            }] addEventAnalyticsHook:OCMArg.any];
```

Comment:
no, I agree with the new terminology

https://github.com/artsy/eigen/blob/master/Artsy/Classes/ARAppDelegate+Analytics.m#L83 is what I though this was